python/graphstorm/utils.py

```diff
@@ -213,8 +237,8 @@ def check(self, name):
         if len(self._checkpoints) >= 2 and self._rank == 0:
             checkpoint1 = self._checkpoints[-2]
             checkpoint2 = self._checkpoints[-1]
-            logging.debug("{}: elapsed time: {:.3f}, mem (curr: {:.3f}, peak: {:.3f}, \
-                    shared: {:.3f}, global curr: {:.3f}, global shared: {:.3f}) GB".format(
+            logging.debug("{}: elapsed time: {:.3f}, mem (curr: {:.3f}, shared: {:.3f}, \
+                    peak: {:.3f}, global curr: {:.3f}, global shared: {:.3f}) GB".format(
```

Comment:
I think the original is correct. Are you sure this is correct?

Reply:
The original one is correct. Let me revert the change.